docs/starting-guide.md

```diff
@@ -0,0 +1,62 @@
+# Starting guide
+
+Oncilla DB is flexible to avoid locking you down in any technology or architecture choices as much as possible. The downside of this is a more involved configuration process.
+
+Install Oncilla DB from npm registry with `npm install --save oncilla`.
+
+Your application data needs to be modeled in terms of kinds. Everything must have a kind and an entity of a particular kind and id will always be synchronized and edited atomically, in full. Everything also has an id, which with Oncilla we generate on the client. For an example, for a to-do app the kinds would be tasks and tags and for ids we can use uuids.
+
+Configure Oncilla DB for your data kinds as described below. You also need to provide your instance of React for Oncilla to generate hooks specific to your application.
+
+```js
+import { configure } from "oncilla/dist/react";
+const {
+  create,
+  useConnectivity,
+  useCreate,
+  useData,
+  useMultipleData,
+  useOncillaDebug,
+  usePendingChanges
+} = configure({
+  data: {
+    categories: {},
+    tasks: {}
+  },
+  React
+});
+```
+
+If you use TypeScript, add a TypeScript type to help the compiler catch typing errors. (The values of the generic parameter object are singular, confusingly)
```

Comment:
>(The values of the generic parameter object are singular, confusingly)

Should we coordinate a change to make these plural if that would make the types easier to understand? Might be easier to do now, rather than when the number of Oncilla adopters has grown.

Reply:
Yeah, you’re right, made a Trello card.